templates/service-template.yml

```
@@ -172,6 +172,11 @@ parameters:
   description: "YAML content containing a map of the node prewarming configuration for each instance type"
   value: "{}"
 
+- name: ADMIN_AUTHZ_CONFIG
+  displayName: Admin API AUTHZ configuration
+  description: "YAML configuration for admin API endpoints authorization"
+  value: "[{method: GET, roles: ['kas-fleet-manager-admin-full', 'kas-fleet-manager-admin-read', 'kas-fleet-manager-admin-write', 'connector-fleet-manager-admin-read', 'connector-fleet-manager-admin-write', 'connector-fleet-manager-admin-full']}, {method:PATCH, roles: ['kas-fleet-manager-admin-full', 'kas-fleet-manager-admin-write', 'connector-fleet-manager-admin-write', 'connector-fleet-manager-admin-full']}, {method:DELETE, roles: ['kas-fleet-manager-admin-full', 'connector-fleet-manager-admin-full']}]"
```

```suggestion
  value: "[{method: GET, roles: ['kas-fleet-manager-admin-full', 'kas-fleet-manager-admin-read', 'kas-fleet-manager-admin-write']}, {method:PATCH, roles: ['kas-fleet-manager-admin-full', 'kas-fleet-manager-admin-write']}, {method:DELETE, roles: ['kas-fleet-manager-admin-full']}]"
```

We could get rid of connectors configuration as this has to be configured on their template as well @valdar @dhirajsb

---

@valdar @dhirajsb @lburgazzoli can you coordinate this on the connectors side?

---

@machi1990 I'm taking a look at all the changes we need to get this to work for connectors.

---

Thanks @dhirajsb 

> 'm taking a look at all the changes we need to get this to work for connectors.

@pawelpaszki will be able to help with that